src/ngraph/runtime/interpreter/int_executable.cpp

```diff
@@ -43,6 +43,9 @@ runtime::interpreter::INTExecutable::INTExecutable(const shared_ptr<Function>& f
     pass::Manager pass_manager;
     pass_manager.register_pass<pass::LikeReplacement>();
     pass_manager.register_pass<pass::FusedOpDecomposition>();
+    // Run this pass for the second time since, some fused operators like LSTMCell may use
+    // other fused operators inside.
```

Comment:
I know it is out of scope of this PR, but what if one of operators will be fused in 3 or more levels?
This should be resolved better than just copy lines of code(in 3 places for now).
It can be merged for now but someone should back to this problem later on.